systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture_thumbnail.url }}" class="profile-pic img-square img-responsive crop_img" alt="{{ user }} profile picture"/>
```

Comment:
`crop_img` is a confusing class name. We are not cropping anything. Moreover do we still need it? The thumbnail picture already set the size of 100x100.